frontend/src/common.js

```diff
@@ -3,8 +3,28 @@ window.Raven.config(window.dsn).install();
 window.dsn = undefined;
 
 const testPilotGA = require('testpilot-ga');
+const {gcmCompliant, sendEvent} = require('./utils');
 window.analytics = new testPilotGA({
   an: 'Firefox Send',
   ds: 'web',
   tid: window.trackerId
 });
+
+gcmCompliant().catch(err => {
+  $('#page-one').attr('hidden', true);
+  $('#download').attr('hidden', true);
+  sendEvent('sender', 'unsupported', {
```

Comment:
Like @pdehaan mentioned above, this metric can't always have the `sender` token. I'll move this back into upload and download and add the respective `sender` and `recipient` tags.